packages/app/src/app/overmind/namespaces/git/actions.ts

```diff
@@ -165,6 +170,18 @@ export const createRepoClicked: AsyncAction = async ({
   }
 };
 
+export const importFromGithub: AsyncAction<string> = async (
+  { state, effects, actions },
+  sandboxUrl
+) => {
+  actions.modalClosed();
```

Comment:
I think this one doesn't work anymore with the new "New Sandbox Modal", since we use the new modal system. Does the modal still close when clicking "Import & Fork"?

Reply:
Do not think so, but we redirect away from it, so it should be good?